src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private static final int MULTIPLIER = 2;
+    private Node<K, V>[] array;
+    private int size;
+    private int threshold;
+
+    MyHashMap() {
+        array = new Node[INITIAL_CAPACITY];
+        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
+        int tempHash = this.hash(key);
+        int index = hashIndex(tempHash);
```

Comment:
```suggestion
        int index = getIndex(key);
```
let's create one method getIndex, witch will calculate hash code and calculate index